extensions/amp-analytics/0.1/test/test-linker.js

```diff
@@ -23,11 +23,13 @@ describe('Linker', () => {
   beforeEach(() => {
     // Linker uses a timestamp value to generate checksum.
     sandbox = sinon.sandbox;
-    sandbox.useFakeTimers();
-    sandbox.stub(Date.prototype, 'getTimezoneOffset').returns(400);
+    sandbox.useFakeTimers(1533329483292);
+    sandbox.stub(Date.prototype, 'getTimezoneOffset').returns(420);
 
     const mockWin = mockWindowInterface(sandbox);
-    mockWin.getUserAgent.returns('Chrome 70');
+    mockWin.getUserAgent.returns('Mozilla/5.0 (X11; Linux x86_64) ' +
```

Comment:
TBH: I don't understand how is this affecting the test below.

Reply:
The parts that I updated in test are "checksum"s, which are computed based on user agents, timestamp, language etc.

https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/linker.js#L33

Reply:
So like it doesn't matter what you put into the stub value? Then what's the reason for the change?

Reply:
to test out the checksum algorithm with a real user agent string. Any other vendors (like Google Analytics) who integrates this feature would like to have the exact same test vectors to make sure identical algorithm.